doc/api/fs.md

```diff
@@ -2004,6 +2004,9 @@ changes:
 Asynchronous rmdir(2). No arguments other than a possible exception are given
 to the completion callback.
 
+*Note*: Using `fs.rmdir()` on a file (not directory) results in an `ENOENT`
+error on Windows and `ENOTDIR` error on UNIX/macOS.
```

Comment:
Nit: I know you already changed it from `Linux` to `UNIX`, but I believe that throughout or docs, we use `POSIX` (and that includes macOS, or at least it does in our docs). 

/cc @nodejs/documentation in case I'm wrong about some aspect of this. :-D